recipes/remove-dagmc-tags/meta.yaml

```diff
@@ -0,0 +1,57 @@
+
+{% set name = "remove_dagmc_tags" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e3e16c6ad219c1764db5db4236d326b4a022b2d62c98475c9318a9e9faa82b72
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
If the only reason you're restricting Windows is a missing dependency, you can still keep it [noarch](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python). During installation, the solver will just inform the user of the missing dependency.

Reply:
Thanks for the tip, 

I was getting some red crosses in the azure pipeline check for windows, so I took a look at the meta.yaml for MOAB which is the main dependency and copied the windows skipping from there.

https://github.com/conda-forge/moab-feedstock/blob/a6cd8ff2cf7835a4a4462d300f8e3d4249fb301a/recipe/meta.yaml#L51

But I can return it to not skip windows if that is the recommended solution.

Reply:
MOAB cannot be noarch because they don't fulfil multiple other conditions (compiler, different tests, etc.) so there, a restriction has to be put in place because conda-forge builds the package for each platform and version which would fail on windows.

But I just saw that you're using `scrips` for the CLI and not `console_scripts entry_points` so the package cannot be noarch, disregard my comment. :)